lib/l10n/intl_en.arb

```
@@ -55,5 +55,144 @@
+  "createdOn": "Created on",
+  "paymentMethodsLabel": "Payment methods",
+  "cancelTrade": "Cancel Trade",
+  "areYouSureCancel": "Are you sure you want to cancel this trade?",
+  "cooperativeCancelMessage": "If you confirm, you will start a cooperative cancellation with your counterparty.",
+  "acceptCancelMessage": "If you confirm, you will accept the cooperative cancellation initiated by your counterparty.",
+  "confirm": "Confirm",
+  "buy": "BUY",
+  "sell": "SELL",
+  "buying": "BUYING",
+  "selling": "SELLING",
+  "creatingNewOrder": "CREATING NEW ORDER",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)